### PR TITLE
Update .travis.yml file

### PR DESCRIPTION
**Pull request by amandaJayanetti**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,8 @@
 
 language: R
-sudo: false
+dist: trusty
+sudo: required
```
Comment (jaylett) [inline .travis.yml]: You can keep using container builds and install packages using the `apt` addon:

```
addons:
  apt:
    packages:
      - libxapian-dev
```

**Comment by jaylett** [comment]
You seem to have been creating a lot of PRs which are almost identical recently, and there's a commit I saw go past that I think you intended to be part of this one but isn't. When I see that you're calling your branches things like 'temp', I suspect you need to spend a little more time on working with git. It's easy to think that it's tedious stuff to get right, and that it takes you away from writing code, but actually if you have a clear plan and are moving carefully, everything tends to become easier. It's also very important when collaborating with others to have excellent use of git and branches, because it's a form of collaboration.

I'm not sure why 03f3979 isn't showing in this PR, so I'll have to comment on it directly — hopefully you'll get my notes. (Let me know if not!)

Please try to always:

 * choose a branch name with care that describes what you're doing
 * create good commits, with good commit messages, even when the changes you're making are tiny
 * don't be tempted to cut corners, again even on tiny changes — that is often when things go wrong

**Comment by amandaJayanetti** [comment]
Sorry, it won't happen again. The only reason I did so was because I was
concerned since the deadline is approaching. I used some branches without
actually meaning to integrate changes committed to those. Just to see if
Travis-CI tests are passing. Will make sure it doesn’t happen again.

**Comment by jaylett** [comment]
@amandaJayanetti don't worry too much about the deadline approaching :) you're doing well, and there's still plenty of time (and you built in some unallocated time at the end of your plan). You don't have to open a new PR to test things; Travis will rerun tests if you push an extra commit to the end of an existing PR.

You probably want to turn off Travis for pushes, so it only tests PRs. (It's in the travis UI somewhere.) That'll just mean you get one check rather than two for each PR (since they're identical, it won't make much difference).

**Comment by amandaJayanetti** [comment]
That's great to hear! :) I turned off travis builds for pushes. I supported
range queries with old version of Xapian as well. Will push that as a
separate PR via dev branch.

**Comment by jaylett** [comment]
This can't be merged until it builds against Xapian 1.2, so you probably want to roll the two together as one PR.

**Pull request closed without merging**